WordPress/src/main/java/org/wordpress/android/ui/domains/DomainRegistrationCheckoutWebViewActivity.kt

```diff
@@ -94,7 +94,7 @@ class DomainRegistrationCheckoutWebViewActivity : WPWebViewActivity(), DomainReg
                 putExtra(URL_TO_LOAD, getCheckoutUrl(input.site))
                 putExtra(AUTHENTICATION_URL, WPCOM_LOGIN_URL)
                 putExtra(REGISTRATION_DOMAIN_NAME, input.domainName)
-                putExtra(REGISTRATION_EMAIL, input.site.email)
+                putExtra(REGISTRATION_EMAIL, input.site?.email)
```

Comment:
Just double-checking if this null-check wouldn't cause a problem in `DomainRegistrationCheckoutWebViewActivity`. I see that by default it's handled as `.orEmpty()` if it's `null`.

Reply:
That's a good question @justtwago 🤔  
I think it is safe to pass a `null` value there since there is the `.orEmpty` check. Functionality wise I noticed that the email field on the checkout page is populated by the user email in the case of the just domain checkout without a site. My understanding is that the email used in that case is the user email.
I'll keep an eye on this as we wrap up [the checkout result](https://github.com/wordpress-mobile/WordPress-Android/issues/19544) in case this creates an issue.